docs/mongocli/command/mongocli-ops-manager-logs-decrypt.txt

```diff
@@ -0,0 +1,88 @@
+.. _mongocli-ops-manager-logs-decrypt:
+
+=================================
+mongocli ops-manager logs decrypt
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Decrypts a log file with the provided local key file or KMIP files.
+
+Syntax
+------
+
+.. code-block::
+
+   mongocli ops-manager logs decrypt [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -f, --file
+     - string
+     - true
+     - Path to the file that contains encrypted audit logs.
+   * - -h, --help
+     - 
+     - false
+     - help for decrypt
+   * - --kmipClientCertificateFile
+     - string
+     - false
+     - Path to the Client Cert file used to connect to the KMIP speaking server.
+   * - --kmipServerCAFile
+     - string
+     - false
+     - Path to the CA file used to connect to the KMIP speaking server.
+   * - --localKeyFile
```

Comment:
Same comments as provided above apply here: 
- KMIP speaking server == server that supports Key Management Interoperability Protocol (KMIP). 
- audit log file
- correct syntax for KEK and LEK on line 54 (as commented for line 54 of another file). 
- the same comment for line 58 as for line 58 in the other file.

Reply:
Updated, but not sure about this bullet point: `* audit log file` 
which is the line that should be updated?